src/molecule/scenario.py

```diff
@@ -194,10 +194,7 @@ def inventory_directory(self) -> str:
         Returns:
             The directory containing the scenario's inventory.
         """
-        if self.config.shared_inventory and not self.config.is_parallel:
-            ephemeral = Path(self.shared_ephemeral_directory)
-        else:
-            ephemeral = Path(self.ephemeral_directory)
+        ephemeral = Path(self.ephemeral_directory)
 
         return str(ephemeral / "inventory")
```

Comment:
Nitpick: the intermediate variable is probably unnecessary now

Reply:
nice catch, i looked right past that